numpy/f2py/capi_maps.py

```diff
@@ -112,7 +114,8 @@
                 'complex_float': 'F',
                 'complex_double': 'D',
                 'complex_long_double': 'D',               # forced casting
-                'string': 'c'
+                'string': 'c',
+                'character': 'c'
```

Comment:
I'm not sure I understand the mapping here, should it be `S` for `NPY_STRING`?

Reply:
This is valid for the old numpy core. The mapping is overwritten below to use `S` as you suggested.